DotNET/Endpoint Examples/Multipart Payload/pdf-with-redacted-text-applied.cs

```diff
@@ -0,0 +1,28 @@
+using System.Text;
+
+using (var httpClient = new HttpClient { BaseAddress = new Uri("https://api.pdfrest.com") })
+{
+    using (var request = new HttpRequestMessage(HttpMethod.Post, "pdf-with-redacted-text-applied"))
+    {
+        request.Headers.TryAddWithoutValidation("Api-Key", "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx");
+        request.Headers.Accept.Add(new("application/json"));
+        var multipartContent = new MultipartFormDataContent();
+
+        var byteArray = File.ReadAllBytes("/path/to/file");
+        var byteAryContent = new ByteArrayContent(byteArray);
+        multipartContent.Add(byteAryContent, "file", "file_name");
```

Comment:
```suggestion
        multipartContent.Add(byteAryContent, "file", "file_name.pdf");
```
Add extension for clarity